config/config.go

```diff
@@ -13,16 +14,18 @@ type Config struct {
 	ReportEventGroup        string        `envconfig:"CONSUMER_GROUP"`
 	Brokers                 []string      `envconfig:"KAFKA_ADDR"`
 	DatasetAPIURL           string        `envconfig:"DATASET_API_URL"`
```

Comment:
Can this shouldnt be logged

Reply:
The url to the dataset API should not have to be hidden